keyboards/hhkb_lite_2/keymaps/via/config.h

```diff
@@ -0,0 +1,3 @@
+#define BOOTMAGIC_LITE_ROW 5
+#define BOOTMAGIC_LITE_COL 9
```

Comment:
These two lines should go in `keyboards/hhkb_lite_2/config.h` instead of here.

The second definition should also be `BOOTMAGIC_LITE_COLUMN`.

```suggestion
```